Operational-Procedures/Control-System-Upgrade/Deployment-Activities/index.rst

```diff
@@ -36,30 +34,32 @@ If changes are necessary to these scripts from work described in the previous se
 
 #. With all the systems OFFLINE, you can log out of your Nubaldo instance as we will clean them up soon.
 #. Once all systems are in OFFLINE, still running CSCs/systems and OSPL daemons need to be cleaned up.
-    #. Get number of currently running daemons/CSCs still in OFFLINE from main OSPL daemon:
-        * *docker exec ospl-daemon grep "federations" durability.log*
+    #. Get number of currently running daemons/CSCs still in OFFLINE from main OSPL daemon: (:ref:`TTS <Deployment-Activities-TTS-Federation-Check>`)
+        * *docker exec ospl-daemon grep "federations" durability.log* 
```

Comment:
Please revert this change and remove the trailing space.

Reply:
Fixed.